media/materials/scripts/lens_flare.compositor

```diff
@@ -3,7 +3,7 @@ compositor CameraLensFlare/Default
   technique
   {
     // Temporary textures
-    texture rt0 target_width target_height PF_A8R8G8B8
+    texture rt0 target_width target_height PF_FLOAT32_RGB
```

Comment:
cc @scpeters - this is somewhat related to issue #2928. We could just change the internal texture format directly if we don't want to have a new sdformat param to configure it.

Reply:
Both issues would benefit from an sdf param because most cases require a smaller, integer texture. Always using a FLOAT32 texture should work for every case but often waste graphics memory.

Reply:
I'm working on a custom SDFormat parameter in https://github.com/scpeters/gazebo/commit/8e88380b5d577894c605ce713593cff82185c0b8, just need to finish writing the test

Reply:
I'm ready to approve all parts of this pull request except for this line. Can we revert it and come back to it later, perhaps in the context of #2928?

Reply:
Agreed. Reverted to PF_A8R8G8B8